airbyte-integrations/connectors/source-db2/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Changelog
+
+## 0.1.0
```

Comment:
this now lives in the docs page, see [this example](https://docs.airbyte.io/integrations/sources/stripe#changelog)

Reply:
Thanks, I've updated